Formula/kubevela.rb

```diff
@@ -18,8 +18,13 @@ class Kubevela < Formula
   depends_on "go" => :build
 
   def install
```

Comment:
Any reason we're not setting `CGO_ENABLED=0` here, as in the [`Makefile`](https://github.com/oam-dev/kubevela/blob/f7ebbf4fa0f036a2d3a9ad90f8adba78f0a804bd/Makefile#L12)?

Reply:
added 
```ruby
ENV["CGO_ENABLED"] = "0"
ENV["GO111MODULE"] = "on"
```